modules/core/shared/src/main/scala/lucuma/core/math/dimensional/measure.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.math.dimensional
+
+import _root_.cats.kernel.Eq
```

Comment:
Why not plain `cats.Eq`?

Reply:
Good catch, must have been an automatic import. It was probably placed below `coulomb._`, so the `_root_` was needed to avoid going into `coulomb.cats`. `OrganizeImports` then moved it to the top.